deploy/docker/clickhouse-setup/docker-compose.yaml

```diff
@@ -186,7 +186,7 @@ services:
       - ../common/nginx-config.conf:/etc/nginx/conf.d/default.conf
 
   otel-collector:
-    image: signoz/signoz-otel-collector:0.66.1
+    image: signoz/signoz-otel-collector:${OTELCOL_TAG:-0.66.1}
```

Comment:
where is env `OTELCOL_TAG` created?

Reply:
It is not created, but included for future when we would have to run specific version combination of signoz and otelcollector.
It would be possible without making changes to the docker-compose .yaml.